src/app/regular-giving/regular-giving.component.html

```diff
@@ -37,6 +37,40 @@
                 <p class="b-rt-0 b-m-0 b-bold">{{ campaign.charity.name }}</p>
               </div>
 
```

Comment:
I don't think we necessarily need to DRY up this code yet, but probably worth adding a comment here mentioning that there's similar in `campaign-info.component.html` and also in `donation-start-container.component.html`, and adding comments to those two as well. So that when we come to edit any of them we're prompted to consider if the other two should be edited at the same time.